test-data/unit/check-flags.test

```diff
@@ -1164,10 +1164,10 @@ implicit_reexport = False
 main:2: error: Module 'other_module_2' has no attribute 'a'
 
 [case testImplicitAnyOKForNoArgs]
-# flags: --disallow-any-generics --show-column-numbers
+# flags: --new-semantic-analyzer --disallow-any-generics --show-column-numbers
 from typing import List
 
 A = List  # OK
-B = List[A]  # E:10: Missing type parameters for generic type
-x: A  # E:4: Missing type parameters for generic type
+B = List[A]  # E:10: Missing type parameters for generic type "list"
+x: A  # E:4: Missing type parameters for generic type "A"
```

Comment:
Here I would ideally expect "A" for both errors, otherwise the first error is a bit confusing. But again, not really important.

Reply:
Indeed, which is half of why that test was failing